TextAnalytics/pitchfork_data/pitchfork_language_model.ipynb

```diff
@@ -0,0 +1,510 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial we are going to implement a transfer learning model for text version of the ULMfit. \n",
```

Comment:
Format this markdown and add TOC with hyperlinks and additional sources to review.